can-auth/vatican.c

```diff
@@ -22,6 +22,7 @@
  */
 #include "vatican.h"
 #include "../drivers/irq_can.h"
+#include "../drivers/compare.c"
```

Comment:
including C files is somewhat evil ;-)

This can be especially dangerous for Sancus, since all trusted functions that may not be inlined have to be annotated with the current SM protection domain(!)